src/shared/flow-types.js

```diff
@@ -5,12 +5,16 @@ import BottomProgress from './common/BottomProgress';
 
 // geographic types
 
+// a single point whose coordinates can be expressed in latitude/longitude
+// in geographic pixel coordinates or in image pixel coordinates
 export type Point = [number, number];
 
 export type Polygon = Array<Point>;
 
 export type BBOX = [number, number, number, number];
```

Comment:
could you add a comment here about which number refers to which positition? e.g. top left, right, east, west.... ;)

Reply:
done!